code/modules/reagents/reagent_containers/hypospray_vr.dm

```diff
@@ -22,3 +22,44 @@
 	..()
 	for (var/i = 1 to 7)
 		new /obj/item/weapon/reagent_containers/hypospray/autoinjector/miner(src)
+
+/obj/item/weapon/reagent_containers/hypospray/science
+	name = "prototype hypospray"
+	desc = "This reproduction hypospray is nearly a perfect replica of the early model DeForest hyposprays, sharing many of the same features. However, there are additional safety measures installed to prevent unwanted injections."
+
+/obj/item/weapon/reagent_containers/hypospray/science/attack(mob/living/M as mob, mob/user as mob)
+	if(!reagents.total_volume)
+		to_chat(user, "<span class='warning'>[src] is empty.</span>")
+		return
```

Comment:
All this copypasta is kinda annoying.

Reply:
No idea how else to do it, since I am overwriting the attack proc.

Reply:
I don't think there _is_ another way.